core/iwasm/compilation/aot_emit_aot_file.c

```diff
@@ -767,7 +776,43 @@ get_init_data_section_size(AOTCompContext *comp_ctx, AOTCompData *comp_data,
 {
     uint32 size = 0;
 
-    size += get_mem_info_size(comp_ctx, comp_data);
+    /*
+     * -------------------------------------------------------
+     * | u32 import_memory_count
+     * -------------------------------------------------------
+     * |                                      |  u8 module_name
+     * |                                      |  u8 field_name
+     * |                                      | u32 flags
+     * | AOTImportMemory[import_memory_count] | u32 num_bytes_per_page
+     * |                                      | u32 init_page_count
+     * |                                      | u32 max_page_count
+     * -------------------------------------------------------
+     * | padding
```

Comment:
Seems there is no padding here, since the previous and the latter are both u32 type?

Reply:
Yes. actually there is not. `align_uint(size, 4)` won't change size. I put it here to match the code pattern below.

Reply:
Got it, thanks.